### ISSUE TITLE
RPC Server Question

### ISSUE THREAD

**Issue by Caveman07** [jpbriquet/docker-decred#7]
I have more of a question than issue.
Does this configuration supports making http requests to the running node?

**Comment by jpbriquet** [comment]
dcrd and dcrwallet daemons expose a JSON-RPC API. The command-line tool "dcrctl" is communicating via HTTP and create JSON-RPC payloads to interact with these daemons. JSON-RPC is a low-level solution.

**dcrctl command line source :**
https://github.com/decred/dcrctl/blob/master/dcrctl.go

It would be much simpler to use the REST API available on the dcrdata project, as it provides many more features to get access/query the blockchain (block, hash, transaction, ...)

**Dcrdata REST API doc:**
https://github.com/decred/dcrdata#dcrdata-api

This API is publicly available on https://explorer.dcrdata.org and you can call it directly. 
Dcrdata can also be started as a private instance and be connected to your dcrd node if you need it.

**Comment by Caveman07** [comment]
I don't think it is viable solution for me as I need to submit raw transaction through http RPC.

**Comment by jpbriquet** [comment]
You can change the listening port (and interface) by addition arguments on the command line when you start dcrd/dcrwallet.

It is all detailed in the official documentation: https://docs.decred.org/wallets/cli/dcrd-and-dcrwallet-cli-arguments/

For dcrd:
--listen

For dcrwallet:
--rpclisten / --grpclisten
